package.json

```diff
@@ -91,47 +91,47 @@
     "express-jwt": "3.4.0",
     "font-awesome": "^4.6.3",
     "graphiql": "0.7.3",
-    "graphql": "0.6.0",
+    "graphql": "0.6.1",
     "graphql-custom-datetype": "^0.3.0",
     "isomorphic-fetch": "2.2.1",
-    "jsonwebtoken": "7.0.1",
+    "jsonwebtoken": "7.1.6",
     "jwt-decode": "^2.0.1",
-    "mailcomposer": "^3.9.0",
+    "mailcomposer": "^3.10.0",
     "mailgun-js": "^0.7.11",
     "ms": "^0.7.1",
     "node-env-flag": "0.1.0",
     "node-uuid": "1.4.7",
     "oy-vey": "^1.0.0-alpha",
-    "piping": "1.0.0-rc.2",
-    "react": "15.1.0",
-    "react-copy-to-clipboard": "^4.1.0",
-    "react-css-modules": "^3.7.6",
-    "react-dom": "15.1.0",
+    "piping": "1.0.0-rc.3",
+    "react": "15.2.1",
```

Comment:
This throws some errors (or at least warnings) for the stuff we already have, but I needed it for the stuff I did.